### ISSUE TITLE
Human- and machine- readable data

### ISSUE THREAD

**Issue by jesusbagpuss** [dcmi/pids_in_dc#8]
As a developer I would like to be able to provide data in a format that both humans and machines can read, to allow simple/lightweight and more complete/complex integrations.

E.g. for data such as: `<dc:creator orcid="0000-0002-8611-8266">Salter, John</dc:creator>`

- simple integration: render as text on website
    Author: Salter, J
- slightly more complex/useful integration: render as linked text on website:
    Author: [Salter, John ![ORCID](https://orcid.org/sites/default/files/images/orcid_16x16.gif)](https://orcid.org/0000-0002-8611-8266)
- more complex: retrieve list of other works related to the same PID
    Author: [Salter, John ![ORCID](https://orcid.org/sites/default/files/images/orcid_16x16.gif)](https://orcid.org/0000-0002-8611-8266)
    - also author of _Paper A_, _Paper B_ and _Paper C_.

**Comment by paulwalk** [comment]
Thanks - this is a very useful laying out of requirement. Would not:
`<dc:creator id="https://orcid.org/0000-0002-8611-8266"` be more in keeping with your requirement raised in #6  (i.e. not specifying orcid in the attribute name, but requiring the orcid host/domain to be expressed in the PID itself)?

**Comment by jesusbagpuss** [comment]
If there are multiple identifiers we need to include (e.g. an ORCID and an ISNI for the same creator), then we either have to have an agreed separator in the id field (whitespace?) or use two attributes?

**Comment by paulwalk** [comment]
Yes - absolutely, this is one of the issues we need to solve. I am writing about this right this moment in a little position paper covering the various issues I hope to get out tomorrow.
